File: backend/src/main/java/com/project/dto/user/response/ProfessionalViewDTO.java
package com.project.dto.user.response;



import java.math.BigDecimal;
import java.time.LocalTime;

import com.project.enums.DayOfWeekEnum;
import com.project.enums.Gender;
import com.project.enums.Specialization;
import com.project.enums.SpokenLanguage;

import lombok.Data;

@Data
public class ProfessionalViewDTO {
	private Long professionalId;
    private Long userId;
	private String fullName;
	private Gender gender;
	 
	 private Specialization specialization;
	 private int experienceYears;
	 private String qualification;
	 private String bio;
	 private BigDecimal consultationFee;
	 private SpokenLanguage spokenLanguage;
	 private boolean isVerified;
	 
	 private Long availabilityId;
	 private DayOfWeekEnum dayOfWeek;
	 private LocalTime startTime;
	 private LocalTime endTime;
}

File: backend_dotnet/SafeMind.Core/DTOs/User/Response/ProfessionalViewDTO.cs
using SafeMind.Core.Enums;

namespace SafeMind.Core.DTOs.User.Response
{
    public class ProfessionalViewDTO
    {
        public long ProfessionalId { get; set; }
        public long UserId { get; set; }
        public string FullName { get; set; } = string.Empty;
        public Gender? Gender { get; set; }
        
        public Specialization Specialization { get; set; }
        public int ExperienceYears { get; set; }
        public string? Qualification { get; set; }
        public string? Bio { get; set; }
        public decimal ConsultationFee { get; set; }
        public SpokenLanguage SpokenLanguage { get; set; }
        public bool IsVerified { get; set; }
    }
}
